packages/hathor-rpc-handler/src/rpcMethods/getAddress.ts

```diff
@@ -106,9 +107,9 @@ export async function getAddress(
         data: {
           address,
         }
-      }, requestMetadata);
+      }, requestMetadata) as AddressRequestConfirmationResponse;
 
-      if (!confirmed) {
+      if (!confirmed.data) {
```

Comment:
suggestion: Since this is a library request, I'd suggest protecting it against a `TypeError` and always return the custom Error class.
```suggestion
      if (!confirmed?.data) {
```